From dd621e25d40a2f5803c5bbdb010b1f89a4eb47cf Mon Sep 17 00:00:00 2001
From: Tobias Schramm <t.schramm@manjaro.org>
Date: Thu, 13 May 2021 16:26:35 +0200
Subject: [PATCH 19/20] regmap: debugfs: allow writing to regmaps from debugfs

This is a bad idea. However it is required, since I have not figured out
how to represent the chained TAS5825M setup of the ottercast to ALSA
properly yet. Also we don't have any dangerous power management devices
in regmap, so this should be fine as a hack for now.

Not-signed-off-by: Tobias Schramm <t.schramm@manjaro.org>
---
 drivers/base/regmap/regmap-debugfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/base/regmap/regmap-debugfs.c b/drivers/base/regmap/regmap-debugfs.c
index 8dfac7f3ed7a..8a45caabe797 100644
--- a/drivers/base/regmap/regmap-debugfs.c
+++ b/drivers/base/regmap/regmap-debugfs.c
@@ -290,7 +290,7 @@ static ssize_t regmap_map_read_file(struct file *file, char __user *user_buf,
 				   count, ppos);
 }
 
-#undef REGMAP_ALLOW_WRITE_DEBUGFS
+#define REGMAP_ALLOW_WRITE_DEBUGFS
 #ifdef REGMAP_ALLOW_WRITE_DEBUGFS
 /*
  * This can be dangerous especially when we have clients such as
-- 
2.31.1

